### DISCUSSION TITLE
Is it possible to change the ratio freely (use the horizontal or vertical lines to squeeze the rect)?

### DISCUSSION THREAD

**Discussion by sheldorC137** [guoyingtao/Mantis#257] [Q&A] [answered]
In the iOS Photo Edit View it is possible to squeeze the cropping rect by dragging the horizontal or vertical line. Is it possible with mantis too?

**Comment by guoyingtao** [comment, marked as answer]
Yes, that is the basic operations of changing cropping rect.
